fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -297,6 +300,8 @@ class TestConfig {
 	bool disableHostname = false;
 	// remote key value store is a child process spawned by the SS process to run the storage engine
 	bool disableRemoteKVS = false;
+	// 7.2 cannot be downgraded to 7.1 or below after enabling encryption-at-rest.
```

Comment:
Thanks for adding this code comment, we `should` put this as part of release notes for sure.